src/CharacterManager.hpp

```diff
@@ -188,12 +190,14 @@ namespace SokuLib
 		void doAnimation();
 	};
 
+    //! @brief Linked list of entities. An entity may hold such a list for projectiles and visual effects notably.
```

Comment:
This is specifically the list of character subobject. These are normally templated. I think they are called CObjectList<character, characterObject> (For Remilia it would be CObjectList<Remilia, RemiliaObject>)